models/cm_default.cpp

```diff
@@ -70,6 +72,7 @@ void
 cm_default::get_status( DictionaryDatum& statusdict ) const
 {
   def< double >( statusdict, names::V_th, V_th_ );
+  def< double >( statusdict, "V_init", V_init_ );
```

Comment:
For consistency reasons, I think it would be better to also define `V_init` as a pre-defined name in `nestkernel/nest_names.{h,cpp}` instead of using the string directly.

Reply:
I have added this to `nest_names.{h,cpp}`. Note that as per @med-ayssar 's suggestion, I have removed the `v_init` argument from all `pre_run_hook()` functions. This means that the initial voltage is now set for each compartment as part of the `compartment_params` dictionary.